### PR TITLE
BaSP-TG-58: Time Sheets list

### PR DESCRIPTION
**Pull request by nicomicheletti**
In this PR the GET and DELETE methods of the list of thimesheets are created. For this, the actions, thunks, reducer and dispatch are created. Finally, the modals are added

### CODE REVIEW

**Review by Julian25 — APPROVED** [review]
Excelente trabajo

**Review by GonzaloBracamonte — APPROVED** [review]
Buenisimo el laburo!

**Review by Evelyn-Heredia — APPROVED** [review]
PR approved!

File: src/redux/timesheets/thunks.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        method: 'DELETE',
+        headers: { 'content-type': 'application/json' }
+      };
+      dispatch(deleteTimesheetPending());
+      const response = await fetch(
+        `${process.env.REACT_APP_API_URL}${resource}/${id}`,
+        requestConfig
+      );
+      const data = await response.json();
+      dispatch(deleteTimesheetSuccess(data.message));
```
Comment (BaltazarCortina1) [inline src/redux/timesheets/thunks.js:41]: Falta validar que la respuesta haya sido exitosa

File: src/Components/TimeSheets/List/index.js [hunk 2]
```diff
@@ -62,24 +46,45 @@ const TimeSheets = () => {
     { header: 'Task', key: 'task' }
   ];
 
+  const confirmationHandler = () => {
+    setConfirmation(false);
+    dispatch(deleteTimesheet(id));
+    !error && dispatch(updateList([...list.filter((timeSheet) => timeSheet._id !== id)]));
```
Comment (BaltazarCortina1) [inline src/Components/TimeSheets/List/index.js:52]: Esto lo deberias hacer en el caso de success del delete

**Pull request merged**